srcs/httpMethod/HTTPMethod.cpp

```diff
@@ -292,37 +288,8 @@ int do_CGI(std::string &response_message_str,
     response_message_str = CreateErrorResponse(500, err_pages); 
     return (500);
   }
-  int cgi_out_stream = cgi.GetCgiSocket();
-
-  /*
-   * Read CGI output
-   *
-   * TODO: implement safe read
-   */
-  ssize_t n;
-  std::stringstream cgi_output;
-  while ((n = read(cgi_out_stream, buf, buf_size)) > 0) {
-    buf[n] = '\0';
-    cgi_output << std::string(reinterpret_cast<const char *>(buf));
-  }
 
-  /*
-   * Create response header

```

Comment:
create response header has been moved to here https://github.com/oooooorriiiii/webserv/blob/c7f0a3e2401879bf35da1f229d0cc43fc3ca2402/srcs/server/server.cpp#L250